frontend/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "ultimate-flashcards",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@mui/icons-material": "^5.15.14",
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.5.2",
```

Comment:
types & jest usually are devDependencies, not normal ones (so they don't need to be installed on slim production build)